CMakeLists.txt

```diff
@@ -54,6 +72,12 @@ link_directories(
   ${OFED_PREFIX}/lib
   ${LEVELDB_PREFIX}/lib
 )
+if(${FREEBSD})
```

Comment:
On 4-7-2016 12:32, Kefu Chai wrote:
> In CMakeLists.txt
> <https://github.com/ceph/ceph/pull/10024#discussion_r69438886>:
> 
>> @@ -54,6 +72,12 @@ link_directories(
>>    ${OFED_PREFIX}/lib
>>    ${LEVELDB_PREFIX}/lib
>>  )
>> +if(${FREEBSD})
> 
> i did read this also, the reason why i'd recommend |if(foo)| is for the
> sake of simplicity and consistency. they are the equivalent under most
> circumstances as explained by the document you quote.

Oke, but then "you|me|others"'d better be pretty sure that the reference
is a "constant", because the errors are otherwise hard to track and
understand.